assets/errata.js

```diff
@@ -13,115 +13,160 @@ a new comment on the issue may be added beginning with the word "Summary".
-- underscore is probably unnecessary...
+/**
+ * Display a single issue with all its comments. The array of comments is used to look for a 'Summary' note; if found, added to the issue display.
+ * 
+ * @param {HTMLElement} node - place to place all the issue description, usually a section element
+ * @param {object} issue - issue object as return by the Github API
+ * @param {object[]} comments - array of comments as returned by the Github API 
+ * @param {string[]} labels - labels assigned to the issue 
+ */
+function display_issue(node, issue, comments, labels) {
+    const display_labels = labels.filter((label) => label !== 'Errata').join(', ');
+    const div = document.createElement('div');
+    div.className = 'issue';
+    node.append(div);
+    div.innerHTML = `<h3>"${issue.title}"</h3>`;
 
-Even more elaborate change: use the octokat library to encapsulate the Github API... that might also be used to control paging size; current code 
-may go wrong if the number of issues is very high...
+    let state = 'Open';
+    let state_class = 'state_open';
+    if (issue.state !== 'open') {
+        state = 'Closed';
+        state_class = 'state_closed';
+    }
 
-*/
+    const pull_request = (issue.pull_request === undefined) ? "No" : "Yes";
+    const p1 = document.createElement('p');
+    p1.innerHTML = `<span class='what'>Issue number:</span> <a href='${issue.html_url}'>#${issue.number}</a><br>`  +
+                    `<span class='what'>Raised by:</span> <a href='${issue.user.url}'>@${issue.user.login}</a><br>` +
+                    `<span class='what'>Extra labels:</span> ${display_labels}<br>`                                 +
+                    `<span class='what'>Pull request? </span> ${pull_request}<br>`                                  +
+                    `<span class='what'>Status:</span> <span class="${state_class}">${state}</span><br> `           ;
+    div.append(p1);
+
+    const p2 = document.createElement('p');
+    p2.innerHTML = `<span class='what'><a href='${issue.html_url}'>Initial description:</a></span> ${issue.body}`;
+    div.append(p2);
 
-$(document).ready(function() {
-    var display_issue = function(node, issue, comments, labels) {
-        var  display_labels = _.reduce(labels, function(memo, label, index) {
-            if( label === "Errata" )
-                return memo
-            else if( memo === "" )
-                return label
-            else
-                return memo + ", " + label
-        }, "");
-        var div = $("<div class='issue'></div>");
-        node.append(div);
-        div.append("<h3>\"" + issue.title + "\"</h3>");
-
-        let state = 'Open';
-        let state_class = 'state_open';
-        if (issue.state !== 'open') {
-            state = 'Closed';
-            state_class = 'state_closed';
+    // See if a summary has been added to the comment.
+    let summary = undefined;
+    comments.forEach((comment) => {
```

Comment:
```suggestion
    comments.forEach(comment => {
```

Reply:
That is a question of style... I usually prefer to keep the arrow function's arguments in brackets, even if there is only one of them.